Example/Podfile

```diff
@@ -5,3 +5,14 @@ target 'DatadogSDKBridge_Example' do
 #  pod 'DatadogSDKObjc', :path => '../../dd-sdk-ios/'
   pod 'DatadogSDKBridge', :path => '../', :testspecs => ['Tests']
 end
+
+post_install do |installer|
+  pwd = Dir.pwd
+
+  # Customize build settings for DatadogSDK (dd-sdk-ios) dependency:
+  dd_sdk_ios_xcconfig = "#{pwd}/Pods/Target Support Files/DatadogSDK/DatadogSDK.debug.xcconfig"
+  File.open(dd_sdk_ios_xcconfig, "a") do |file|
+    file.puts("\n// Override build settings for DatadogSDK:")
+    file.puts("#include \"../../../xcconfigs/DatadogSDK.debug.xcconfig\"")
+  end
```

Comment:
can we directly set `SWIFT_ACTIVE_COMPILATION_CONDITIONS` here? that would be easier to follow/read
example from [Cocoapods guides](https://guides.cocoapods.org/syntax/podfile.html#post_install):
```ruby
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['SWIFT_ACTIVE_COMPILATION_CONDITIONS'] = '$(inherited) DD_SDK_COMPILED_FOR_TESTING'
    end
  end
end
```

Reply:
Done 👌. The difference is that now changing configuration requires `pod install` and changes `Podfile.lock`, but I don't think we will be adding more stuff any soon.